pubsub/cloud-client/quickstart.py

```diff
@@ -19,76 +19,110 @@
 
 
 def end_to_end(project_id, topic_name, subscription_name, num_messages):
-    # [START pubsub_end_to_end]
+    # [START pubsub_quickstart_end2end_basic]
```

Comment:
Could you please keep a version of the old sample with the same tag (pubsub_end_to_end) alongside this one, to minimize docs breakages.  Maybe rename the function in the old sample?

Reply:
The old region tag hasn't been used in the docs yet. So it is still safe to decide what name we want. I will revert to the old one `pubsub_end_to_end`.